validator/webui/index.html

```diff
@@ -456,6 +456,11 @@
           button.addEventListener('click', function() {
             var urlToFetch = input.value.trim();
             if (urlToFetch.length > 0) {
+              if (!(urlToFetch.indexOf('http://') == 0 ||
+                    urlToFetch.indexOf('https://') == 0)) {
+                urlToFetch = 'https://' + urlToFetch;
```

Comment:
I think it's OK to be suggestive with https. If it turns out that the resource is http only, the user can edit it. Since they're a webmaster - they really *should* make it work on https anyway.